plugins/modules/aci_epg_to_contract_interface.py

```diff
@@ -0,0 +1,325 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1', 'status': ['preview'], 'supported_by': 'certified'}
+
+
+DOCUMENTATION = r'''
+---
+module: aci_epg_to_contract_interface
+short_description: Bind EPGs to Consumed Contracts Interface (fv:RsConsIf).
+
+description:
+- Bind EPGs to Consumed Contracts Interface on Cisco ACI fabrics.
+notes:
+- The C(tenant), C(app_profile), C(EPG), and C(Contract) used must exist before using this module in your playbook.
+  The M(cisco.aci.aci_tenant), M(cisco.aci.aci_ap), M(cisco.aci.aci_epg), and M(cisco.aci.aci_contract) modules can be used for this.
+options:
+  tenant:
+    description:
+    - Name of an existing tenant.
+    type: str
+    aliases: [ tenant_name ]
+  ap:
+    description:
+    - Name of an existing application profile, that will contain the EPGs.
+    type: str
+    aliases: [ app_profile, app_profile_name ]
+  epg:
+    description:
+    - The name of the end point group.
+    type: str
+    aliases: [ epg_name ]
+  contract:
```

Comment:
I would call it contract_interface and maybe put contract in the alias and keep contract_interface_name also in the alias.